src/index.js

```diff
@@ -1,2 +1,61 @@
-/* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
+/* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension, no-use-before-define */
```

Comment:
```suggestion
/* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
```
`no-use-before-define`을 disable 하지 않고 코드를 작성하려면 어떻게 해야할까요? 한번 고민해보시면 좋을것 같습니다~

Reply:
no-use-before-define 룰을 disable 하지 않는 부분이 존재하군요. :)
이 부분 해결해보려고 고민해 보았는데, 어떻게 해야 할지 도저히 떠오르지 않아서 그냥 disable 했는데 ㅎㅎ

한번 disable 하지 않는 방향으로 적용해 보겠습니다.

Reply:
https://eslint.org/docs/rules/no-use-before-define 이 문서를 참고하시면 왜 이런 룰이 생겼는지 도움이 되실 것 같습니다 💯